backend_jobs/ingestion_pipeline/setup.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
where/how is this file used?

Reply:
when we upload to data flow we need to have a zip file with the packages we will use,
to create ta zip file we use the setup file.

(we need also the requirement file with all the different imports when we upload to data flow )